galgebra/ga.py

```diff
@@ -1838,7 +1835,6 @@ def __init__(self, *kargs, **kwargs):
         u = kargs[0]  # Coordinate map or vector embedding to define submanifold
         coords = kargs[1]  # List of cordinates
         ga = kwargs['ga']  # base geometric algebra
-        self.wedge_print = kwargs['wedge']
```

Comment:
This line was a clear copy-paste bug and this self.wedge_print turns out to be not used and useless for this class.